src/utils.ts

```diff
@@ -293,14 +294,47 @@ export function getEnsTextRecord(ens: string, record: string, network = '1') {
   return call(provider, ENS_RESOLVER_ABI, [address, 'text', [ensHash, record]]);
 }
 
-export async function getSpaceUri(id, network = '1') {
+export async function getSpaceUri(
+  id: string,
+  network = '1'
+): Promise<string | null> {
   try {
     return await getEnsTextRecord(id, 'snapshot', network);
   } catch (e) {
-    return false;
+    console.log(e);
+    return null;
   }
 }
 
+export async function getEnsOwner(ens: string): Promise<string | null> {
```

Comment:
So this function returns the ENS controller. not owner/registrant 😄 we accidentally solved our problem?

Reply:
Example try to call this function for: https://app.ens.domains/name/abc.eth/details

Reply:
Okay then if that the case I guess the controller is fine :D

Reply:
it was not reading controllers from goerli. fixed it